### ISSUE TITLE
Find original source of odd line in Queues file

### ISSUE THREAD

**Issue by alodahl** [Techtonica/curriculum#802]
We got come feedback on our [Queues outline](https://github.com/Techtonica/curriculum/blob/master/data-structures/queues.md):
There's a line "Queue (high volume architecture - processing events sequentially)" but that parenthetical does not refer to the data structure. Was there another line item for the high volume architecture kind of Queue that got accidentally merged into this Queue?  

## Task:
**Solve this mystery!**

Go into https://github.com/Techtonica/curriculum/blob/master/data-structures/queues.md and find the line they mention.  Is the real reference somewhere in the history of this file?  

Report back via the comments in this issue. Once your task is complete, fix a single typo or whitespace mistake in any file, make a pull request with it, but with the PR title and description describing this issue instead. This way you can get your Hacktoberfest contribution.

**Comment by tiffanyjstuart** [comment]
Is the line "Queue (high volume architecture - processing events sequentially)" supposed to appear somewhere in the current outline or slides? I'm not seeing it...

**Comment by alodahl** [comment]
@tiffanyjstuart thanks for pointing out my postgresql goof.

I have no clue about the reference... I was hoping it would be in the file history somewhere.  If theres not evidence at all, we should just delete that part.

**Comment by tiffanyjstuart** [comment]
@alodahl I don't see that line is anywhere in the file, so it don't think it needs to be deleted. I think you can just close this Issue, unless you have a pointer to the specific line.

**Comment by alodahl** [comment]
Thanks Tiffany
